graphql/resolve/mutation_rewriter.go

```diff
@@ -1648,6 +1703,18 @@ func rewriteObject(
 	return frag, upsertVar, retErrors
 }
 
+func xidErrorForInterfaceType(typ schema.Type, xidString string, xidName,
+	interfaceName string) error {
+	if queryAuthSelector(typ) == nil {
+		// This error will only be reported in debug mode.
+		return x.GqlErrorf("interface %s; field %s: id %s already exists for one of the implementing"+
+			" type of interface", interfaceName, xidName, xidString)
+
+	}
+	return x.GqlErrorf("GraphQL debug: interface %s; field %s: id %s already exists for "+
```

Comment:
The error message was probably changed to make it consistent with this error message. I feel the old error message can also be used with interface. "id %s already exists for field %s inside one of the implementing types of interface %s "